bin/MadGraph5_aMCatNLO/cards/production/13TeV/WJets_HT_LO_MLM/WJetsToQQ_HT-600toInf/WJetsToLNu_HT-600toInf_proc_card.dat

```diff
@@ -0,0 +1,25 @@
+import model sm-ckm_no_b_mass
+#switch to diagonal ckm matrix if needed for speed
+#import model sm-ckm_no_b_mass
+
+define p = p b b~
+define j = j b b~
+define q = j b b~
```

Comment:
This is confusing, since the b's are being added twice.  I assume madgraph is smart enough to remove duplicates from the definition?